src/screens/EditCaretakerScreen.tsx

```diff
@@ -13,19 +13,19 @@ const EditCaretakerScreen = ({route}:NativeStackScreenProps<RootStackParamList,
 
   // WIP
   const {info} = route.params;
-
+  
   const { t } = useTranslation();
   return (
     <KeyboardAvoidingView>
       <ScrollView>
         <View bgColor="#FAFAFA">
-          <ProfileInfoCard fname={info["fname"]} lname={info["lname"]} gender="Female" bdate="08/02/1917" phone="090909090"/>
+          <ProfileInfoCard fullName={info["fullName"]} gender={info["gender"]} bdate={info["bdate"]} phone={info["phone"]} imageId={info["imageId"]}/>
```

Comment:
it would be nice to use info.fullName

change the type of /navigation/types.ts in the screen to the one you pass into this route